code/LeftAndMain.php

```diff
@@ -1540,6 +1540,12 @@ public function getEditForm($id = null, $fields = null)
             $readonlyFields = $form->Fields()->makeReadonly();
             $form->setFields($readonlyFields);
         }
+
+        // Sudo mode
```

Comment:
Scenario is editing an individual record that is a record managed by a gridfield e.g. CMSProfileController will use this when editing the logged in users profile